io.catenax.shared.shopfloor_information_types/2.0.0/ShopfloorInformationTypes.ttl

```diff
@@ -0,0 +1,85 @@
+##########################################################################################
+# Copyright (c) 2023-2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IOSB & Fraunhofer IWU & Fraunhofer IPA)
+# Copyright (c) 2023-2024 Siemens AG
+# Copyright (c) 2023-2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+##########################################################################################
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.shared.shopfloor_information_types:2.0.0#> .
+@prefix ext-header: <urn:samm:io.catenax.shared.message_header:2.0.0#> .
+
+:ShopfloorInformationTypes a samm:Aspect ;
```

Comment:
any reason for making aspect name plural? it does not have any List/Set as root element.

Reply:
The defined aspect is a shared aspect that is not used by us. However the namespace features two types that we re-use in our other models: 

the property communicationMode, and the TimeValueCharacteristic. Thats the reason for the plural.

If the plural is still a problem i can also change it to singular, Would that change also mean that we have to update the name of our namespace to io.catenax.shared.shopfloor_information_type ?